index.js

```diff
@@ -167,12 +167,10 @@ export const validateReceiptIos = async (receiptBody, isTest) => {
   if (!response.ok) {
     throw Object.assign(new Error(response.statusText), { statusCode: response.status })
   }
-
-  return response.json();
-};
+}
```

Comment:
@dooboolab it seems like the merge commit removed the return value, submitting a PR for this soon 👌